reporter.js

```diff
@@ -3,8 +3,53 @@ var throughObj = require('through2').obj
 
 const PLUGIN_NAME = 'gulp-pug-linter'
 
-module.exports = function (hasToFail) {
+var defaultReporter = function (errors) {
```

Comment:
This project is using [jsDoc](http://usejsdoc.org/) for annotations. To make it easier for myself and others when reviewing and using the code, I like to make sure that every function is annotated.

What do you think of the following annotation for this?

```js
/**
 * @name defaultReporter
 * @description Logs all errors via `gulp-util`
 * @param {Array} errors Pug lint error objects with `message`
 */
```